src/software/ai/hl/stp/play/example/example_play_fsm.h

```diff
@@ -0,0 +1,56 @@
+#pragma once
+
+#include "proto/parameters.pb.h"
+#include "shared/constants.h"
+#include "software/ai/hl/stp/play/play.h"
+#include "software/ai/hl/stp/tactic/move/move_tactic.h"
+#include "software/logger/logger.h"
+
+/**
+ * An example Play that moves the robots in a circle around the ball
+ */
+
+struct ExamplePlayFSM
+{
+    class MoveState;
+
+    struct ControlParams
+    {
+    };
+
+    DEFINE_PLAY_UPDATE_STRUCT_WITH_CONTROL_AND_COMMON_PARAMS
+
+    /**
+     * Creates an example play FSM
+     *
+     * @param ai_config the play config for this play FSM
+     */
+    explicit ExamplePlayFSM(const TbotsProto::AiConfig& ai_config);
```

Comment:
This play FSM doesn't use `TbotsProto::AiConfig`, so you probably can get rid of the `ai_config` field and just use the default constructor

Reply:
you removed `ExamplePlayFSM` from the `.cpp` file but you also need to remove it from the `.h` file. Right now, this mismatch causes linker errors whenever you compile it.